inst/qml/qml_components/LSestimationpriorandposterior.qml

```diff
@@ -0,0 +1,285 @@
+						inclusive:		JASP.MinMax
+					}
+
+					DoubleField
+					{
+						visible:		plotsPriorIndividualType.currentText == "custom"
+						enabled:		plotsPriorIndividualCI.checked
+						name:			"plotsPriorUpper"
+						label:			qsTr("Upper")
+						id:				plotsPriorUpper
+						fieldWidth:		50
+						defaultValue:	if(analysisType == "binomial"){ 0.75 }else{ 1 }
+						min:			plotsPriorLower.value
+						max:			if(analysisType == "binomial"){ 1 }else{ 9999999999 }
+						inclusive:		JASP.MinMax
+					}
+				}
+			}
+		}
+	}
+
+	CheckBox
+	{
+		name:		"plotsPosterior"
+		label:		qsTr("Posterior distribution")
+		checked:	false
+
+		RadioButtonGroup
+		{
+			name: 	"plotsPosteriorType"
+
+			RadioButton
+			{
+				value:		"overlying"
+				label:		qsTr("All")
+				checked:	true
+			}
+
+			RadioButton
+			{
+				value:		"stacked"
+				label:		qsTr("Stacked")
+			}
+
+			RadioButton
+			{
+				value:	"individual"
+				label:	qsTr("Individual")
+
+				CheckBox
+				{
+					label:				qsTr("Point estimate")
+					name:				"plotsPosteriorIndividualEstimate"
+					childrenOnSameRow:	true
+
+					DropDown
+					{
+						name:	"plotsPosteriorIndividualEstimateType"
+						label:	""
+						values:	["mean", "median", "mode"]
+					}
+				}
+
+				CheckBox
+				{
+					name:				"plotsPosteriorIndividualCI"
+					label:				qsTr("CI")
+					id:					plotsPosteriorIndividualCI
+					childrenOnSameRow:	true
+
+					DropDown
+					{
+						name:		"plotsPosteriorIndividualType"
+						label:		""
+						values:		["central", "HPD", "custom", "support"]
+						id:			plotsPosteriorIndividualType
+					}
+				}
+
+				Group
+				{
+					columns: 2
+
+					CIField
+					{
+						visible:		plotsPosteriorIndividualType.currentText == "central" | plotsPosteriorIndividualType.currentText == "HPD"
+						enabled:		plotsPosteriorIndividualCI.checked
+						name:			"plotsPosteriorCoverage"
+						label:			qsTr("Mass")
+						fieldWidth:		50
+						defaultValue: 	95
+						min:			0
+						max:			100
+						inclusive:		JASP.MaxOnly
+					}
+
+					DoubleField
+					{
+						visible:		plotsPosteriorIndividualType.currentText == "custom"
+						enabled:		plotsPosteriorIndividualCI.checked
+						name:			"plotsPosteriorLower"
+						label:			qsTr("Lower")
+						id:				plotsPosteriorLower
+						fieldWidth:		50
+						defaultValue:	if(analysisType == "binomial"){ 0.25 }else{ -1 }
+						min:			if(analysisType == "binomial"){ 0 }else{ -9999999999 }
+						max:			plotsPosteriorUpper.value
+						inclusive:		JASP.MinMax
+					}
+
+					DoubleField
+					{
+						visible:		plotsPosteriorIndividualType.currentText == "custom"
+						enabled:		plotsPosteriorIndividualCI.checked
+						name:			"plotsPosteriorUpper"
+						label:			qsTr("Upper")
+						id:				plotsPosteriorUpper
+						fieldWidth:		50
+						defaultValue:	if(analysisType == "binomial"){ 0.75 }else{ 1 }
+						min:			plotsPosteriorLower.value
+						max:			if(analysisType == "binomial"){ 1 }else{ 9999999999 }
```

Comment:
same as above (I see a repeating theme here, does it make sense to make this a qml component)